Description: According to DPP 5.1 all perl programs must start with
             "#!/usr/bin/perl"
Author: Elimar Riesebieter <riesebie@lxtec.de>
Last-Update: 2010-02-22.

Index: ripit-3.9.0/ripit.pl
===================================================================
--- ripit-3.9.0.orig/ripit.pl	2010-09-25 17:56:29.425816169 +0200
+++ ripit-3.9.0/ripit.pl	2010-09-25 17:56:40.392058043 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 ########################################################################
 #
 # LICENSE
